RecoEgamma/EgammaPhotonProducers/python/conversionOpenTrackSequence_cff.py

```diff
@@ -0,0 +1,18 @@
+import FWCore.ParameterSet.Config as cms
+
+import RecoEgamma.EgammaPhotonProducers.conversionTrackProducer_cfi
+
+#producer from lowPt electronTracksOpen
+gsfTracksOpenConversionTrackProducer = conversionTrackProducer.clone(
```

Comment:
Ok, since above you don't use the "import ... from" syntax, the full path name should be restored. sorry for the wrong suggestion I gave.!
In any case, please test your code with the short matrix before pushing the commits, so that you can verify yourself whether it works or not and speed up the integrtion

Reply:
The way it was before it was extra tested. I did not test your suggestion out of trust